data/json/recipes/recipe_others.json

```diff
@@ -2313,5 +2313,24 @@
       [ [ "glass_shard", 3 ], [ "pipe_glass", 1 ], [ "flask_glass", 3 ], [ "test_tube", 6 ], [ "marble", 75 ] ],
       [ [ "stopcock", 1 ] ]
     ]
+  },
+  {
+    "result": "training_dummy_light",
+    "type": "recipe",
+    "category": "CC_OTHER",
+    "subcategory": "CSC_OTHER_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 2,
+    "time": "1 h",
+    "autolearn": true,
+    "proficiencies": [ { "proficiency": "prof_carpentry_basic" } ],
+    "qualities": [ { "id": "HAMMER", "level": 2 }, { "id": "SAW_W", "level": 2 } ],
+    "components": [
+      [ [ "wood_panel", 1 ] ],
+      [ [ "2x4", 2 ] ],
+      [ [ "nail", 30 ] ],
+      [ [ "duct_tape", 50 ] ],
+      [ [ "2x4", 2 ], [ "log", 1 ], [ "wood_beam", 1 ], [ "stick_long", 1 ] ]
```

Comment:
a training dummy made of 1 long stick or 2 planks would be very flimsy compared to one made from a log

Reply:
I did the recipe based in what @keampe suggested, I can remove the long stick, but with planks maybe just add 2 more?